src/components/TodosFilter.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import { FilterType } from '../types/FIlterType';
```

Comment:
```suggestion
import { FilterType } from '../types/FilterType';
```
There is a typo in file's name. Change 'I' to 'i'